web/styles/right_sidebar.css

```diff
@@ -448,6 +461,7 @@ $user_status_emoji_width: 24px;
     grid-template-columns: minmax(0, 1fr) auto;
     align-items: center;
     margin-bottom: 10px;
+    margin-left: 5px;
```

Comment:
Should this 5px be a variable?

Reply:
I ended up not using a variable, because I wanted it to line up with the left arrow. Using 0px or the width of the arrow both seemed wrong.

<img width="259" alt="image" src="https://github.com/user-attachments/assets/677d5883-4eec-4f50-892d-8b4f62e3d31d">

<img width="269" alt="image" src="https://github.com/user-attachments/assets/3c87d05e-f4f5-495c-b635-0388d2e34a67">

Reply:
OK. Probably the sort of thing where I'd have written a comment.

Reply:
Ah okay, good to know!